spec/active_interaction/concerns/runnable_spec.rb

```diff
@@ -242,6 +242,26 @@ def execute
         expect(outcome).to be_valid
       end
     end
+
+    context 'with failing composition' do
+      class CheckInnerForFailure
+        include ActiveInteraction::Runnable
+
+        attr_reader :caught_error
+
+        def execute
+          compose(WrappableFailingInteraction)
+        rescue => interrupt
```

Comment:
`Interrupt` is technically a private error within ActiveInteraction. I don't see a reason to make the specific error available. Since that's the case I'm not catching the specific error type here.